### DISCUSSION TITLE
这里可以有瑟瑟吗

### DISCUSSION THREAD

**Discussion by coolwide** [share121/inter-knot#5701] [公告]
![1725955298240](https://github.com/user-attachments/assets/72ba177b-6d1e-411e-823a-d4e49580467b)
可以在这里有瑟瑟吗，会不会被封捏，大家有没有想看的哇🥵

**Comment by hot-ice-seele** [comment]
不清楚，毕竟版规都忘了长啥样了

**Reply by 123acnl** [reply]
又发现你了(￣_,￣ )

**Comment by KELERSAMA** [comment]
传奇绳网用户 一夜发几十条 好几条色情内容被封了 还被挂起来了

**Comment by 123acnl** [comment]
理论上不可以

**Comment by Yizakl** [comment]
#4628

**Comment by YolandaWang-0** [comment]
都没了一次了，还是老实点吧，说不定明天就上不了了。

**Comment by 129776** [comment]
太过头的不行，色图不能发，好像是这样
![Uploading 1731329281345.gif…]()

**Comment by Hoyoer** [comment]
![1732869698583.gif](https://github.com/user-attachments/assets/06571b4d-21cf-4711-bdb9-d309fcf4ed91)

你想看哪种的
